src/app/static/src/tests/components/runWorkflow/runWorkflowCreate.test.ts

```diff
@@ -28,18 +67,86 @@ describe(`runWorkflowCreate`, () => {
         expect(wrapper.emitted("create").length).toBe(1)
     })
 
-    it(`can emit run navigation step`, async() => {
+    it(`can emit run navigation step`, async (done) => {
         const wrapper = getWrapper()
-        expect(wrapper.find("h2").text()).toBe("Run workflow")
-        await wrapper.find("#rerun").trigger("click")
-        expect(wrapper.emitted("rerun").length).toBe(1)
+
+        setTimeout(async () => {
+            expect(wrapper.find("h2").text()).toBe("Run workflow")
+            expect(mockAxios.history.get.length).toBe(1)
+            expect(mockAxios.history.get[0].url).toBe("http://app/workflows")
+            expect(wrapper.vm.$data.error).toStrictEqual("")
+            expect(wrapper.vm.$data.defaultMessage).toStrictEqual("")
+            expect(wrapper.vm.$data.workflows).toStrictEqual(workflowSummaryMetadata)
+
+            const vueSelect = wrapper.find(VueSelect)
+            vueSelect.vm.$emit("input", selectedWorkflow)
```

Comment:
I am not sure it's a best practice to test library functionality.

Reply:
Unless I've missed something this comment still holds: we need a test that the dropdown is showing some workflows, and that they're correctly formatted.

We also need a Selenium test that ensures that this page can be navigated to and that a workflow can be created. This test can then be extended as we develop other stages of the wizard.

Reply:
Sure, do you know how to test vue-select dropdown with the formatted list ? Also there are no populated existing workflow items for selenium test to pass, it will fail on buildkite.

Reply:
> Sure, do you know how to test vue-select dropdown with the formatted list ?

Should just be a case of finding the relevant elements - a bit like what we do for the autocomplete widget used for reports e.g. https://github.com/vimc/orderly-web/blob/master/src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunReportPageTests.kt#L57

> Also there are no populated existing workflow items for selenium test to pass, it will fail on buildkite.

You can insert into the database in Selenium test... see existing one here: https://github.com/vimc/orderly-web/blob/master/src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt